src/components/form/README.md

```diff
@@ -249,6 +249,16 @@ The select checkmark icon can be defined not only via `actions` slot but also vi
 </ui-select>
 ```
 
+You can also override the default select icon once for all via `@dreipol/vue-ui/src/settings`:
+
+```js
+import settings, {SELECT_ICON} from '@dreipol/vue-ui/src/settings';
```

Comment:
Can we adapt this to also use the code style of our project?
`import settings, { SELECT_ICON } from '@dreipol/vue-ui/src/settings';`

Reply:
updated